pynapple/io/neurosuite.py

```diff
@@ -394,9 +397,8 @@ def read_neuroscope_intervals(self, name):
             print("reading intervals...")
             evt_file = os.path.join(self.path, self.path.split('/')[-1] + "." + name + ".evt")
```

Comment:
It should be self.basename and not dependant on the path

Reply:
I would do something like this : 
```
def read_neuroscope_intervals(self, pathtofile=None, name=None):
      if pathtofile is None and name is not None:
             pathtofile = os.path.join(self.path, self.basename + "." + name + ".evt")
     else:
            raise RuntimeError('Please specifiy a path or name")
```

Reply:
Alright, (;